src/DonationForms/Properties/FormSettings.php

```diff
@@ -262,6 +262,20 @@ class FormSettings implements Arrayable, Jsonable
      */
     public $currencySwitcherSettings;
 
+    /**
+     * Form Tags are a custom taxonomy. This setting is only used to pass the selected terms to the server.
+     * @unreleased
+     * @var array
+     */
+    public $formTags;
+
+    /**
+     * Form Categories are a custom taxonomy. This setting is only used to pass the selected terms to the server.
+     * @unreleased
+     * @var array
+     */
+    public $formCategories;
+
```

Comment:
Can you explain your comment here "This setting is only used to pass the selected terms to the server"? 

The form settings object stores all values into the database as json.  Is that an unwanted side effect?

Reply:
It was an imperfect solution, but I decided to go with your suggestion to pull the data from `$_POST`. See [558e723](https://github.com/impress-org/givewp/pull/7463/commits/558e723908fa1e8975fd3bc511b5b0dae76e2a47).